test/lib/type/string_test.exs

```diff
@@ -45,6 +46,21 @@ defmodule Antikythera.StringTypesTest do
     refute Domain.valid?("end-with-hyphen-")
     refute Domain.valid?("-" <> String.duplicate("a", 63))
     refute Domain.valid?(String.duplicate("a", 63) <> "-")
+    # should be disjoint from CowboyWildcardSubdomain
+    refute Domain.valid?(":_.example.com")
+  end
+
+  test "validate CowboyWildcardSubdomain" do
+    assert CowboyWildcardSubdomain.valid?(":_.example.com")
+
+    refute CowboyWildcardSubdomain.valid?(":_.")
+    refute CowboyWildcardSubdomain.valid?("*.example.com")
+    refute CowboyWildcardSubdomain.valid?(":subdomain.example.com")
+    refute CowboyWildcardSubdomain.valid?("subdomain.:_.example.com")
```

Comment:
I would like to test that the waildcard cannot be used more than once.
```suggestion
    refute CowboyWildcardSubdomain.valid?("subdomain.:_.example.com")
    refute CowboyWildcardSubdomain.valid?(":_.:_.example.com")
```

Reply:
Thank you for your suggestion.
I applied it at https://github.com/access-company/antikythera/pull/170/commits/e5b9f524dfdf3cc661d9bfc1660850902fd50ea7